### ISSUE TITLE
Anti-Neurofilaments antibody positivity

### ISSUE THREAD

**Issue by MaudDLVT** [obophenotype/human-phenotype-ontology#10504]
**Preferred term label:**
Anti-Neurofilaments antibody positivity

**Synonyms**
NfL-IgG positive

**Definition (free text, please give PubMed ID)**
The presence of autoantibodies in the serum that react against neuronal intermediate filament
PMID: 30282771

**Parent term (use hpo.jax.org/app)**
HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:624259

**Your nano-attribution (ORCID)**
0000-0002-3387-1836

**Comment by pnrobinson** [comment]
@MaudDLVT 
Adding new term: 
- ID: HP:6001058
- label: Anti-neuronal intermediate filament antibody positivity
- parents: [Autoimmune antibody positivity (HP:0030057)](https://hpo.jax.org/app/browse/term/HP:0030057)
- pmid: PMID:30282771|PMID:30282771
- def: The presence of autoantibodies (immunoglobulins) in the blood circulation that react against neuronal intermediate filament.
- comment: 
- orcid: orcid:0000-0002-3387-1836
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10504
- synonyms:
